src/studio/kdb/Config.java

```diff
@@ -212,24 +214,42 @@ public void setNotesHash(String notesHash) {
         save();
     }
 
-    public Font getFont() {
-        String name = p.getProperty("font.name", "Monospaced");
-        int  size = Integer.parseInt(p.getProperty("font.size","14"));
+    private Font getFont(String prop) {
+        String name = p.getProperty(prop + ".name", "Monospaced");
+        int size = 14;
+        try {
+            size = Integer.parseInt(p.getProperty(prop + ".size", ""));
+        } catch (NumberFormatException e) {
+            // do nothing
```

Comment:
I would prefer to have here log.error. This indicates that either we have a bug or someone manually modified and corrupted the file